"use strict";(self.webpackChunkgatsby_starter_clean_resume=self.webpackChunkgatsby_starter_clean_resume||[]).push([[506],{6486:function(e,t,n){var a=n(6540);t.A=e=>a.createElement("a",{href:e.link,className:"badge"},e.children)},106:function(e,t,n){n.r(t),n.d(t,{default:function(){return d}});var a=n(6540),i=n(8669),s=n(6486);var o=e=>{const t=e.badges.map(((t,n)=>a.createElement(s.A,{key:e.key+"_"+n,link:t.link},t.name)));return a.createElement("table",{className:"item-table"},a.createElement("tbody",null,a.createElement("tr",null,a.createElement("td",{className:"item-name"},e.name||""," ",t),a.createElement("td",{className:"item-when"},e.when||"")),a.createElement("tr",null,a.createElement("td",{className:"item-org",colSpan:"2"},e.description||""))))},r=n(7626),l=n(4848),m=(0,r.A)((0,l.jsx)("path",{d:"M9.4 16.6 4.8 12l4.6-4.6L8 6l-6 6 6 6zm5.2 0 4.6-4.6-4.6-4.6L16 6l6 6-6 6z"}),"Code"),c=JSON.parse('[{"name":"dyspyosis","description":"Python package to detect dysbiosis in microbiome data","year":2024,"URLs":[{"name":"GitHub","link":"https://github.com/raeslab/dyspyosis"}]},{"name":"RLdbRDA","description":"RaesLab distance-based redundancy analysis","year":2024,"URLs":[{"name":"GitHub","link":"https://github.com/raeslab/RLdbRDA"}]},{"name":"loreplotr","description":"Logistic Regression Plots for R","year":2024,"URLs":[{"name":"GitHub","link":"https://github.com/raeslab/loreplotr"}]},{"name":"lorepy","description":"Logistic Regression Plots for Python","year":2023,"URLs":[{"name":"GitHub","link":"https://github.com/raeslab/lorepy"},{"name":"Zenodo","link":"https://zenodo.org/record/8324902"}]},{"name":"CoNekT","description":"Co-Expression Network Toolkit","year":2018,"URLs":[{"name":"GitHub","link":"https://github.com/sepro/CoNekT"}]},{"name":"PhytoNet","description":"Comparative co-expression network analyses across phytoplankton and land plants","year":2018,"URLs":[{"name":"Link","link":"http://www.gene2function.de"}]},{"name":"LSTrAP","description":"Large-Scale Transcriptomics Analysis Pipeline","year":2017,"URLs":[{"name":"GitHub","link":"https://github.com/sepro/LSTrAP"}]},{"name":"Plasmid DB","description":"Lab Management software","year":2015,"URLs":[{"name":"Link","link":"http://sebastian.proost.science/plasmid-db/"}]},{"name":"PLAZA","description":"Online comparative genomics platform","year":2014,"URLs":[{"name":"Link","link":"http://bioinformatics.psb.ugent.be/plaza/"}]},{"name":"Trapid","description":"Online transcript annotation and phylogeny tool","year":2013,"URLs":[{"name":"Link","link":"http://bioinformatics.psb.ugent.be/webtools/trapid/"}]},{"name":"i-ADHoRe","description":"Software to detect genomic homology","year":2012,"URLs":[{"name":"GitHub","link":"https://github.com/VIB-PSB/i-ADHoRe"}]}]'),p=JSON.parse('[{"name":"statannotations","description":"Python package to add annotations to seaborn plots","year":2023,"URLs":[{"name":"GitHub","link":"https://github.com/trevismd/statannotations"},{"name":"Zenodo","link":"https://zenodo.org/record/8396665"}]}]');var d=()=>{const e=c.map(((e,t)=>a.createElement(o,{key:t,name:e.name,when:e.year,description:e.description,badges:e.URLs}))),t=p.map(((e,t)=>a.createElement(o,{key:t,name:e.name,when:e.year,description:e.description,badges:e.URLs})));return a.createElement(i.A,{pageTitle:"Software",pageDescription:"Learn more about things I developped",showTitle:!0},e,a.createElement("h3",null,a.createElement(m,{className:"media-icon"})," Open Source Contributions"),t)}}}]);
//# sourceMappingURL=component---src-pages-software-js-5177528605e1a901f90d.js.map